.github/workflows/build.yml

```diff
@@ -251,10 +301,20 @@ jobs:
           export LD_LIBRARY_PATH="/root/workspace/mmdeploy/build/lib:${LD_LIBRARY_PATH}"
           python3 tools/check_env.py
       - name: Test TensorRT pipeline
+        id: badge_status
```

Comment:
why add id here?

Reply:
> why add id here?
Get the status of the step before badge step to know if previous steps are all success or not.